### PR TITLE
Fixes related to open issues.

### LINKED ISSUE #27: Refer to DesignerSerializationVisibility attribute
When it comes to generate efficient code, the [`DesignerSerializationVisibility` attribute](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.designerserializationvisibilityattribute) would be worth to be mentioned. Depending on the scenario, the attribute fits better.

An example:

My scenario was a custom inherited `ToolStripButton`. It introduces a new property which controls some of the properties of the base classes. Those properties should no longer be serialized.

I first tried the methodology described in this page before finding out that the `DesignerSerializationVisibility` attribute works best for me, using the following pattern:

```C#
[Browsable(false)]
[DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
public new bool CheckOnClick { get => base.CheckOnClick; set => base.CheckOnClick = value; }
```

By the way, there are two swapped words "a have" in the first sentence.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e32682fb-671a-8f6e-045c-e92a96ca029e
* Version Independent ID: 42210b30-f3ee-9975-b3f2-8752f9aca815
* Content: [Defining Default Values with the ShouldSerialize and Reset Methods - Windows Forms .NET Framework](https://docs.microsoft.com/en-us/dotnet/desktop/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods?view=netframeworkdesktop-4.8)
* Content Source: [dotnet-desktop-guide/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md](https://github.com/dotnet/docs-desktop/blob/master/dotnet-desktop-guide/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md)
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### LINKED ISSUE #47: Not all triggerbase derivations mentioned as method of starting Storyboard
_Ported from dotnet/docs. Created by @tonyhallett_

Table fails to mention other applicable TriggerBase derivations MultiDataTrigger and and MultiTrigger.

Here is a MultiTrigger [example](https://docs.microsoft.com/en-us/dotnet/api/system.windows.multitrigger?view=netframework-4.7.2#examples) and a MultiDataTrigger [example](https://docs.microsoft.com/en-us/dotnet/api/system.windows.multidatatrigger?view=netframework-4.7.2#examples)

Also, the attached property Storyboard.TargetName is mentioned but not Storyboard.Target which is useful when targeting a static resource.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5bbf3b44-d30c-f80e-de53-830a309343b9
* Version Independent ID: 7a6ca3f1-31e9-9cfd-0469-c477e8888233
* Content: [Storyboards Overview - WPF .NET Framework](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/graphics-multimedia/storyboards-overview?view=netframeworkdesktop-4.8#how-to-apply-animations-with-a-storyboard)
* Content Source: [dotnet-desktop-guide/framework/wpf/graphics-multimedia/storyboards-overview.md](https://github.com/dotnet/docs-desktop/blob/master/dotnet-desktop-guide/framework/wpf/graphics-multimedia/storyboards-overview.md)
* Product: **dotnet-framework**
* Technology: **dotnet-wpf**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### LINKED ISSUE #61: Intro to framework winforms page mentions core
https://docs.microsoft.com/en-us/dotnet/desktop/winforms/?view=netframeworkdesktop-4.8

The intro is more generic than it should be. This page is for .NET Framework only.

### LINKED ISSUE #62: Code example comment lists Border instead of ScrollViewer
In the first code example on this page, the last comment reads
"// Add the Border as the Content of the Parent Window Object"
I think it should not be "Border" but "ScrollViewer" like this.
// Add the ScrollViewer as the Content of the Parent Window Object

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 712dcc2a-20de-b5be-a9cf-2b548f76768e
* Version Independent ID: c4e96199-5cbd-f9ef-ba03-261614d02ffd
* Content: [ScrollViewer Overview - WPF .NET Framework](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/scrollviewer-overview?view=netframeworkdesktop-4.8)
* Content Source: [dotnet-desktop-guide/framework/wpf/controls/scrollviewer-overview.md](https://github.com/dotnet/docs-desktop/blob/master/dotnet-desktop-guide/framework/wpf/controls/scrollviewer-overview.md)
* Product: **dotnet-framework**
* Technology: **dotnet-wpf**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### LINKED ISSUE #67: just add link to list of characters...
just add link to list of characters...
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8089bee4-feef-f0a5-1691-120bfa240784
* Version Independent ID: 595f2172-fa4a-0d3e-03b0-516ca73f41cc
* Content: [How to: Use Special Characters in XAML - WPF .NET Framework](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/advanced/how-to-use-special-characters-in-xaml?view=netframeworkdesktop-4.8)
* Content Source: [dotnet-desktop-guide/framework/wpf/advanced/how-to-use-special-characters-in-xaml.md](https://github.com/dotnet/docs-desktop/blob/master/dotnet-desktop-guide/framework/wpf/advanced/how-to-use-special-characters-in-xaml.md)
* Product: **dotnet-framework**
* Technology: **dotnet-wpf**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### LINKED ISSUE #83: DesiredHeight property in Style doesn't exist on Border control
[Enter feedback here]
                            <Border x:Name="**Content**" Grid.Row="1" BorderThickness="1,0,1,1" CornerRadius="0,0,2,2">
                                <Border.BorderBrush>
                                    <SolidColorBrush Color="{DynamicResource BorderMediumColor}" />
                                </Border.BorderBrush>
                                <Border.Background>
                                    <SolidColorBrush Color="{DynamicResource ContentAreaColorDark}" />
                                </Border.Background>
                                <ContentPresenter Margin="4" />
                            </Border>
                        </Grid>
                        <ControlTemplate.Triggers>
                            <Trigger Property="IsExpanded" Value="True">
                                <Setter TargetName="ContentRow" Property="Height" Value="{Binding **DesiredHeight**, ElementName=**Content**}" />
                            </Trigger>
                        </ControlTemplate.Triggers>

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bdb9feb2-67b1-f850-9acb-38f05f9ce5f1
* Version Independent ID: 1ad823ca-dbdf-a145-0265-0cf29056ec98
* Content: [Expander Styles and Templates - WPF .NET Framework](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/expander-styles-and-templates?view=netframeworkdesktop-4.8)
* Content Source: [dotnet-desktop-guide/framework/wpf/controls/expander-styles-and-templates.md](https://github.com/dotnet/docs-desktop/blob/master/dotnet-desktop-guide/framework/wpf/controls/expander-styles-and-templates.md)
* Product: **dotnet-framework**
* Technology: **dotnet-wpf**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Adjust landing page for .NET Framework winforms.
- Add tip about the DesignerSerializationVisibilityAttribute attribute.
- Add MultiTrigger and MultiDataTrigger to WPF overview.
- Fix some comments in a code snippet.
- Add some spacing in get started table.
- Add missing single quote character in XAML info and link to reference.
- Change DesiredHeight to Height on XAML style.

Fixes #61
Fixes #27
Fixes #47
Fixes #62
Fixes #67
Fixes #83

### CODE REVIEW

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-use-special-characters-in-xaml.md [hunk 1]
```diff
@@ -11,21 +11,23 @@ helpviewer_keywords:
...
+The following table shows the syntax for encoding this set of special characters:
+
+| Character | Syntax   | Description          |  |
+|-----------|----------|----------------------|--|
+| <         | `&lt;`   | Less than symbol.    |  |
+| >         | `&gt;`   | Greater than sign.   |  |
+| &         | `&amp;`  | Ampersand symbol.    |  |
+| "         | `&quot;` | Double quote symbol. |  |
+| '         | `&apos;` | Single quote symbol. |  |
+
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-use-special-characters-in-xaml.md:25]: Changes were between here, rest of file was killing off the doublespaces at ends of lines.

File: dotnet-desktop-guide/framework/winforms/index.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ### YamlMime:Landing
 
 title: .NET Desktop Guide for Windows Forms
-summary: Learn about using Windows Forms on Windows with either .NET Framework, .NET 5 and above, or .NET Core 3.1.
+summary: Learn about using Windows Forms (WinForms), a graphical user interface for Windows and .NET Framework.
```
Comment (IEvangelist) [inline dotnet-desktop-guide/framework/winforms/index.yml:4]: Not sure how I feel about "Learn about using", consider either "Learn how to use ..." or "Learn about ..."

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-use-special-characters-in-xaml.md [hunk 3]
```diff
@@ -11,21 +11,23 @@ helpviewer_keywords:
...
+The following table shows the syntax for encoding this set of special characters:
+
+| Character | Syntax   | Description          |  |
+|-----------|----------|----------------------|--|
+| <         | `&lt;`   | Less than symbol.    |  |
+| >         | `&gt;`   | Greater than sign.   |  |
+| &         | `&amp;`  | Ampersand symbol.    |  |
+| "         | `&quot;` | Double quote symbol. |  |
+| '         | `&apos;` | Single quote symbol. |  |
```
Comment (IEvangelist) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-use-special-characters-in-xaml.md]:
```suggestion
| Character | Syntax   | Description          |
|-----------|----------|----------------------|
| `<`       | `&lt;`   | Less than symbol.    |
| `>`       | `&gt;`   | Greater than sign.   |
| `&`       | `&amp;`  | Ampersand symbol.    |
| `"`       | `&quot;` | Double quote symbol. |
| `'`       | `&apos;` | Single quote symbol. |
```

**Review by IEvangelist — APPROVED** [review]
A few nits

**Comment by adegeo** [comment]
Verified that the snippets5000 error is related to old snippet that I'm not going to port.

**Pull request merged**